TemplateMethod/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Epam.NetMentoring.Adapter.TemplateMethod
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
it would be nice to have a bit more comprehensive tests, I mean - we could cover a bit more cases, i.e.:
- valid/not valid, at least check results, etc.

Reply:
ok